.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "8"
+  - "6"
+  - "4"
+
+services:
+  - redis-server
+
+before_install:
+  - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
+  - npm cache clean
```

Comment:
I'd be tempted now to change this to check the version to be equal to 2 rather than not equal 3, since otherwise it'll downgrade npm@5 to 3, which we don't want to do.